sdk/batch/azure-compute-batch/src/test/java/com/azure/compute/batch/FileTests.java

```diff
@@ -100,7 +100,7 @@ public void canReadFromNode() throws Exception {
             if (waitForTasksToComplete(batchClient, jobId, TASK_COMPLETE_TIMEOUT_IN_SECONDS)) {
                 BatchTask task = batchClient.getTask(jobId, taskId);
                 String nodeId = task.getNodeInfo().getNodeId();
-                PagedIterable<NodeFile> files = batchClient.listNodeFiles(poolId, nodeId, null, null, null, null, true);
+                PagedIterable<NodeFile> files = batchClient.listNodeFiles(poolId, nodeId, null, null, null, true);
```

Comment:
See the other comment on using the options bag syntax since the whole point of them was to avoid having to pass `null`s as positional arguments.

Reply:
Same as above comment - this change is related to removing the ocp-date header and has nothing to do with the options bag change.